### ISSUE TITLE
ImportError: cannot import name 'LatentDirichletAllocation'

### ISSUE THREAD

**Issue by emtropyml** [IndicoDataSolutions/finetune#453]
```
ImportError                               Traceback (most recent call last)
<ipython-input-1-435d22fa4e7e> in <module>
      6 from sklearn.preprocessing import MultiLabelBinarizer
      7 from sklearn.metrics import roc_auc_score
----> 8 from finetune import MultiLabelClassifier
      9 from finetune.base_models import BERT, BERTLarge, GPT2, GPT2Medium, GPT2Large, TextCNN, TCN, RoBERTa, DistilBERT
     10 from finetune.config import Settings

~/.local/lib/python3.5/site-packages/finetune/__init__.py in <module>
      9 from tensorflow.compat.v1 import logging as tf_logging
     10 
---> 11 from finetune.target_models.multifield import MultiFieldClassifier, MultiFieldRegressor
     12 from finetune.target_models.classifier import Classifier
     13 from finetune.target_models.regressor import Regressor

~/.local/lib/python3.5/site-packages/finetune/target_models/multifield.py in <module>
      2 
      3 from finetune.errors import FinetuneError
----> 4 from finetune.target_models.classifier import Classifier, ClassificationPipeline
      5 from finetune.target_models.regressor import Regressor, RegressionPipeline
      6 from finetune.base import BaseModel

~/.local/lib/python3.5/site-packages/finetune/target_models/classifier.py in <module>
      3 import tensorflow as tf
      4 import numpy as np
----> 5 from imblearn.over_sampling import RandomOverSampler
      6 from sklearn.utils import shuffle
      7 

~/.local/lib/python3.5/site-packages/imblearn/__init__.py in <module>
     32     Module which allowing to create pipeline with scikit-learn estimators.
     33 """
---> 34 from . import combine
     35 from . import ensemble
     36 from . import exceptions

~/.local/lib/python3.5/site-packages/imblearn/combine/__init__.py in <module>
      3 """
      4 
----> 5 from ._smote_enn import SMOTEENN
      6 from ._smote_tomek import SMOTETomek
      7 

~/.local/lib/python3.5/site-packages/imblearn/combine/_smote_enn.py in <module>
      8 from sklearn.utils import check_X_y
      9 
---> 10 from ..base import BaseSampler
     11 from ..over_sampling import SMOTE
     12 from ..over_sampling.base import BaseOverSampler

~/.local/lib/python3.5/site-packages/imblearn/base.py in <module>
     14 from sklearn.utils.multiclass import check_classification_targets
     15 
---> 16 from .utils import check_sampling_strategy, check_target_type
     17 
     18 

~/.local/lib/python3.5/site-packages/imblearn/utils/__init__.py in <module>
      5 from ._docstring import Substitution
      6 
----> 7 from ._validation import check_neighbors_object
      8 from ._validation import check_target_type
      9 from ._validation import check_sampling_strategy

~/.local/lib/python3.5/site-packages/imblearn/utils/_validation.py in <module>
     11 
     12 from sklearn.base import clone
---> 13 from sklearn.neighbors._base import KNeighborsMixin
     14 from sklearn.neighbors import NearestNeighbors
     15 from sklearn.utils.multiclass import type_of_target

~/.local/lib/python3.5/site-packages/sklearn/neighbors/__init__.py in <module>
     15 from ._kde import KernelDensity
     16 from ._lof import LocalOutlierFactor
---> 17 from ._nca import NeighborhoodComponentsAnalysis
     18 from ._base import VALID_METRICS, VALID_METRICS_SPARSE
     19 

~/.local/lib/python3.5/site-packages/sklearn/neighbors/_nca.py in <module>
     20 from ..base import BaseEstimator, TransformerMixin
     21 from ..preprocessing import LabelEncoder
---> 22 from ..decomposition import PCA
     23 from ..utils.multiclass import check_classification_targets
     24 from ..utils.random import check_random_state

~/.local/lib/python3.5/site-packages/sklearn/decomposition/__init__.py in <module>
     17 from ._factor_analysis import FactorAnalysis
     18 from ..utils.extmath import randomized_svd
---> 19 from ._online_lda import LatentDirichletAllocation
     20 
     21 __all__ = ['DictionaryLearning',

ImportError: cannot import name 'LatentDirichletAllocation'
```

**Comment by benleetownsend** [comment]
This seems like the same issue as #450 can @madisonmay advise on the version that imblearn needs pinning too?

**Comment by benleetownsend** [comment]
This should have been fixed by #454 and will be included in the next release. If you need a fix sooner then you'll need to pin the versions to the ranges shown in this PR. I will close this issue for now, feel free to re-open if the problem persists.
